script/update_gossip_nodes.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+include GossipNodeLogic
+
+%w[mainnet testnet].each do |network|
+  logger = Logger.new("#{Rails.root}/log/gossip_nodes_#{network}.log")
+
+  payload = {
+    config_urls: Rails.application.credentials.solana["#{network}_urls".to_sym],
+    network: network
+  }
+
+  p = Pipeline.new(200, @payload)
+              .then(&get_nodes)
+              .then(&update_nodes)
+              .then(&set_staked_flag)
+
+  logger.info "Finished at #{DateTime.now} with status #{p.code}"
+
+  if p.errors
+    logger.error p.errors.message
+    p.errors.backtrace.each do |line|
+      logger.error line
```

Comment:
Maybe it would be better to do one error with the whole backtrace, `logger.error p.errors.backtrace`. Current code will cause printing of date and time near every line from backtrace nad it might be a bit noisy.

Reply:
if it's done all at once there's no whitespaces and all the backtrace is a oneliner